libs/employer/feature/dashboard/src/lib/components/employer-sub-category/employer-sub-category.component.html

```diff
@@ -1,7 +1,9 @@
 <ng-container *ngIf="loadSubCategory$ | async as subCategory; else loading">
   <section class="sub-category-container">
     <div class="sub-category-header">
-      <h2 class="sub-category-title" *ngIf="loadTransactionDetails$ | async as transaction; else label">{{ transaction.transactionDefinitionName }}</h2>
+      <ng-container *ngIf="!subCategory.displayLabel; else label">
```

Comment:
we are displaying the label in both versions, we can use a better name. this will require updates in several places I believe

```suggestion
      <ng-container *ngIf="!subCategory.useTransactionLabel; else label">
```